### ISSUE TITLE
what is difference between pyramid_flow_miniflux_bf16_v1.safetensors and pyramid_flow_miniflux_bf16_v2.safetensors ? 

### ISSUE THREAD

**Issue by ptits** [kijai/ComfyUI-PyramidFlowWrapper#36]
and any chance to have fp32 version or it does not make sense?

**Comment by kijai** [comment]
I wish I knew that myself, the v1 was the first one they uploaded, then they replaced it silently after 12 hours, the results are drastically different with them so I kept them both. For fp32 v2 you can use the transformer from their repo.

**Comment by ptits** [comment]
heh, it is funny - they point to you when i asked them :)

"This is kijai's version https://github.com/kijai/ComfyUI-PyramidFlowWrapper

It is suggested that you should ask kijai."

**Comment by kijai** [comment]
> heh, it is funny - they point to you when i asked them :)
> 
> "This is kijai's version https://github.com/kijai/ComfyUI-PyramidFlowWrapper
> 
> It is suggested that you should ask kijai."
> 
> from here: https://github.com/jy0205/Pyramid-Flow/issues/145

Yes we'll you should ask why the initial  model was replaced instead then.

**Comment by ptits** [comment]
> For fp32 v2 you can use the transformer from their repo.

do you mean this:
https://huggingface.co/rain1011/pyramid-flow-miniflux/tree/main/diffusion_transformer_384p

**Comment by kijai** [comment]
> > For fp32 v2 you can use the transformer from their repo.
> 
> do you mean this:
> https://huggingface.co/rain1011/pyramid-flow-miniflux/tree/main/diffusion_transformer_384p
> 

Yeah, that should be it, if you look at the commit history there it shows they replaced it too. My "V2" is that current file as bf16
